ui/lib/core/addon/templates/components/replication-dashboard.hbs

```diff
@@ -9,16 +9,41 @@
         data-test-isReindexing />
     </div>
   {{/if}}
-  <div
-    class="selectable-card-container {{if isSecondary 'secondary' 'primary'}}"
-    data-test-selectable-card-container={{if isSecondary 'secondary' 'primary'}}>
-    {{yield (hash
-      card=(component componentToRender replicationDetails=replicationDetails)
-    )}}
-    {{#if (not isSecondary)}}
-      {{yield (hash secondaryCard=(component 'known-secondaries-card'))}}
-    {{/if}}
-  </div>
+  {{#if isSummaryDashboard}}
+    <div class="summary-state">
+      <h6 class="title is-5 {{if (get (cluster-states summaryState) "isOk") "" "has-text-danger"}}" data-test-error>state</h6>
+      {{#unless (get (cluster-states summaryState) "isOk")}}
+        <AlertInline
+          @type="danger"
+          @message="The cluster is {{summaryState}}. Please check your server logs." />
+      {{else}}
+        <p class="has-text-grey is-size-8">The current operating state of both Disaster Recovery and Performance. This value should be ‘running.’ If the value is idle, it indicates an issue and needs to be investigated.</p>
+      {{/unless}}
+      <h2 class="title is-3" data-test-summary-state>{{summaryState}}
+        {{#if (get (cluster-states summaryState) "isOk")}}
+          <Icon @glyph={{get (cluster-states summaryState) "glyph"}} class="{{if (get (cluster-states summaryState) "isOk") "has-text-success" "has-text-danger"}}" data-test-icon />
+        {{/if}}
+      </h2>
+    </div>
+    <div
+      class="selectable-card-container summary secondary"
+      data-test-selectable-card-container-summary>
+        {{yield (hash
+          card=(component componentToRender replicationDetails=replicationDetailsSummary)
```

Comment:
i wonder if in the case of the summary dashboard we should just hardcode the `componentToRender` here -- it's a little confusing to have to go back and find out what component we're passing in here. the downsides of contextual components (ᵟຶ︵ ᵟຶ)

Reply:
Hmm.. I agree, but I also don't like that this would be a one-off, where we use componentToRender elsewhere.  I think I would rather change all of them, instead of just the one, or keep it as is.  Thoughts?

Reply:
yeah, that totally makes sense! i agree that it'd be better to change all of 'em vs just this one. in a future refactor it might be nice to update the dashboard template to something like:

``
if summary dashboard
  show summary stuff
else if primary dashboard
  show primary stuff
else if secondary stuff
  show secondary stuff
```

this would eliminate some of the abstraction (and also flexibility) within the dashboard template which could be helpful for debugging. just something to consider for later if you think it'd help & we have time.